README.md

```diff
@@ -113,22 +113,6 @@ On Plone 5, Volto is currently tested with the following packages pinned to spec
 - plone.rest 2.0.0a5
 - plone.volto 4.0.0a13
 
-This would be the docker command to spawn a Plone 5 container with the right KGS versions:
-
-```shell
-docker run -it --rm --name=plone -p 8080:8080 -e SITE=Plone -e ADDONS="plone.restapi==8.30.0 plone.volto==4.0.0a13 plone.rest==2.0.0a5 plone.app.iterate==4.0.2 plone.app.vocabularies==4.3.0" -e PROFILES="plone.volto:default-homepage" plone/plone-backend
```

Comment:
plone.restapi==8.39.0
The relations service is introduced in 8.39.0
https://github.com/plone/plone.restapi/commit/880e98321b0120954c9b9ab88c9ea689df5ad0ce

Reply:
@ksuess GH just showed me your comment :( I will update the README in other PR